sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -933,25 +941,35 @@ public Query apply(Function parent, Function inner, Context context) {
                         break;
                     case LteOperator.NAME:
                         includeUpper = true;
+                        includeLower = true;
```

Comment:
Can you please create another PR with only the fix against master with changes entry? We'll have to backport this to 0.54 and 0.55 as well.